libraries/AP_Notify/AP_Notify.h

```diff
@@ -93,6 +103,10 @@ class AP_Notify
     // handle a LED_CONTROL message
     static void handle_led_control(mavlink_message_t* msg);
 
+    static const struct AP_Param::GroupInfo var_info[];
+
+    AP_Int8 _rgb_led_brightness;
```

Comment:
_ data members need to be private.  Once you make this private the RGBLed class still needs to access it so you can either make AP_Notify a friend class OR use get/set methods.